pkg/trace/api/normalizer_test.go

```diff
@@ -218,6 +220,17 @@ func TestNormalizeStartTooSmall(t *testing.T) {
 	assert.True(t, s.Start > before, "start should have been reset to current time")
 }
 
+func TestNormalizeStartTooSmallWithLargeDuration(t *testing.T) {
```

Comment:
Can we at the very least merge the `TestNormalizeStart` tests and make the rest subtests? e.g.:
```go
    t.Run("pass", ...)
    t.Run("small", ...)
    t.Run("large-duration", ...)
```

Reply:
Ok, updated